test/extensions/filters/http/router/config_test.cc

```diff
@@ -32,7 +32,7 @@ TEST(RouterFilterConfigTest, SimpleRouterFilterConfig) {
   RouterFilterConfig factory;
   Http::FilterFactoryCb cb = factory.createFilterFactoryFromProto(proto_config, "stats.", context);
   Http::MockFilterChainFactoryCallbacks filter_callback;
-  EXPECT_CALL(filter_callback, addStreamDecoderFilter(_));
+  EXPECT_CALL(filter_callback, addStreamDecoderFilter(_)).Times(1);
```

Comment:
`.Times(1)` is redundant, as this is already implied by `EXPECT_CALL`. I'd remove these changes.

Reply:
removed